### PR TITLE
add alternative_string_arrange method

### PR DESCRIPTION
**Pull request by suryapratapsinghsuryavanshi**
### **Describe your change:**

* [x] Add an algorithm?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by Sin-Sumit — APPROVED** [review]
alternative_string_arramgements.py is reviewed by me and it is good.

File: strings/alternative_string_arrange.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    'AXBYZ'
+    >>> alternative_string_arrange("ABC", "")
+    'ABC'
+    """
+    first_str_length: int = len(first_str)
+    second_str_length: int = len(second_str)
+    abs_length: int = (
+        first_str_length if first_str_length > second_str_length else second_str_length
+    )
+    li: list = []
```
Comment (L3str4nge) [inline strings/alternative_string_arrange.py:21]: I don't like name `li`. If you name it for example `output_list` would be more readable.

**Review by L3str4nge — CHANGES_REQUESTED** [review]
Please change variable name and I will approve it because it looks good :)

File: strings/alternative_string_arrange.py (hunk 1 above)
Comment (suryapratapsinghsuryavanshi) [reply strings/alternative_string_arrange.py:21]: Sure, I change it.

**Review by L3str4nge — APPROVED** [review]

**Pull request merged**